src/index.js

```diff
@@ -0,0 +1,4 @@
+import ObjectTable, { ObjectCell, ObjectRow, BaseEditor, TextEditor, TextDrawer } from './object-table'
+
+export default ObjectTable
+export { ObjectCell, ObjectRow, BaseEditor, TextEditor, TextDrawer }
```

Comment:
Is this change tackling [this comment](https://github.com/uptick/react-object-list/pull/125#issuecomment-851876404)? If it is, maybe add a comment linking these PRs for posterity

Reply:
Yes it does tackle it and for sure I will :)